libs/datamodel/core/tests/attributes/default_positive.rs

```diff
@@ -55,12 +54,6 @@ fn should_set_default_for_all_scalar_types() {
         .assert_default_value(DefaultValue::new_single(PrismaValue::Json(
             r#"{ "a": ["b"] }"#.to_owned(),
         )));
-    user_model
-        .assert_has_scalar_field("decimal")
-        .assert_base_type(&ScalarType::Decimal)
-        .assert_default_value(DefaultValue::new_single(PrismaValue::Float(
-            r#"121.10299000124800000001"#.parse().unwrap(),
-        )));
```

Comment:
This test doesn't appear to be about mongo, it should still work, does it need to change?

Reply:
How do I exclude Mongo from running this then?

Reply:
It only runs once, without a connector, so it's never run with a mongo datasource.

Reply:
Ah, it fails because Decimal is now a capability. The empty connector has no capabilities, so no Decimal support.

Reply:
Ah I see, we could save that test by adding a datasource with any provider that has decimals, then